src/main/java/appeng/client/gui/implementations/GuiPatternTerm.java

```diff
@@ -41,13 +42,20 @@
 public class GuiPatternTerm extends GuiMEMonitorable
 {
 
-	final ContainerPatternTerm container;
+	private final static String SUBSITUTION_DISABLE = "0";
+	private final static String SUBSITUTION_ENABLE = "1";
 
-	GuiTabButton tabCraftButton;
-	GuiTabButton tabProcessButton;
-	// GuiImgButton substitutionsBtn;
-	GuiImgButton encodeBtn;
-	GuiImgButton clearBtn;
+	private final static String CRAFTMODE_CRFTING = "1";
+	private final static String CRAFTMODE_PROCESSING = "0";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)